spec/support/feature_helpers.rb

```diff
@@ -32,7 +32,7 @@ def choose_school(school)
     click_button "Continue"
 
     # flaky test workaround in case the first click on Continue submitted the form
-    click_button "Continue" unless /(claim|current)-school\?_method=patch/.match?(current_url)
+    click_button "Continue" unless /(claim|current)-school\?_method=get/.match?(current_url)
```

Comment:
Is this related to the changes in the pr or is it just a test fix?

Reply:
It shouldn't have been there; I was playing with a test during the rebase. Corrected, thanks